scripts/tests/test_iso_name_tagging.py

```diff
@@ -266,28 +257,25 @@ def test_main_intersect_sam_file_id(
         with open(output, "r") as out_file:
             assert captured.out == out_file.read()
 
-    def test_main_invalid_intersect_file(
-        self, monkeypatch, capsys, invalid_file
-    ):
+    @pytest.mark.parametrize("lines", ["8", "13"])
```

Comment:
I think it's better to just enumerate the entire filenames here. Why? Imagine you change, for some reason, the filenames of the test files. Or you want to add another one. You would probably search for any instances where the filename is used. But you'd almost certainly miss it, because what you really need to search for is "8" and "13". And then, if a new or renamed file doesn't follow the naming pattern, you not only have to change the parameterization, but the actual code in the test method as well.